cookbook/tests/api/test_api_cook_log.py

```diff
@@ -33,21 +33,21 @@ def test_list_permission(arg, request):
 
 
 def test_list_space(obj_1, obj_2, u1_s1, u1_s2, space_2):
-    assert len(json.loads(u1_s1.get(reverse(LIST_URL)).content)) == 2
-    assert len(json.loads(u1_s2.get(reverse(LIST_URL)).content)) == 0
+    assert json.loads(u1_s1.get(reverse(LIST_URL)).content)['count'] == 2
+    assert json.loads(u1_s2.get(reverse(LIST_URL)).content)['count'] == 0
 
     obj_1.space = space_2
     obj_1.save()
 
-    assert len(json.loads(u1_s1.get(reverse(LIST_URL)).content)) == 1
-    assert len(json.loads(u1_s2.get(reverse(LIST_URL)).content)) == 0
+    assert json.loads(u1_s1.get(reverse(LIST_URL)).content)['count'] == 1
+    assert json.loads(u1_s2.get(reverse(LIST_URL)).content)['count'] == 1
 
 
 @pytest.mark.parametrize("arg", [
     ['a_u', 403],
-    ['g1_s1', 404],
+    ['g1_s1', 403],  # changed expected value.  based on list permissions the log is visible, but not editable
     ['u1_s1', 200],
-    ['a1_s1', 404],
+    ['a1_s1', 403],  # changed expected value.  based on list permissions the log is visible, but not editable
```

Comment:
interesting, this is definitely an oversight on my part because i do not think any user should be able to see logged cooking events of other users or maybe this could be explicitly added as a sharing feature in the future. I will fix this after merging